voyager_archive/templates/voyager_archive/marc_display.html

```diff
@@ -11,18 +11,33 @@
 </form>
 <hr/>
 
-{% comment %}
-Raw Marc dump
-{% endcomment %}
+<div class="row">
+    <div class="column">
+        <h2>Record Leader: {{ marc_record.leader }}</h1>
+        <table class="data-display">
+            <tr>
+                <th>Tag</th>
+                <th>Indicators</th>
+                <th>Data and Subfields</th>
+            </tr>
+            {% for field in marc_record.fields %}
+            <tr>
+                <td>{{ field.tag }}</td>
+                <td><span class='indicators'>{{ field.indicators }}</span></td>
+                <td>
+                    {{ field.data }}
+                    {% for delimiter, values in field.subfields.items %}
+                        <span class='subfield'>${{ delimiter }}</span>
+                        {% for value in values %}
+                            {{ value }}
```

Comment:
This needs to be reworked.  At present it displays only the values for each subfield - which is fine if a subfield occurs only once in a field, but wrong when a subfield occurs multiple times in a field.

Using bib 9411434 as an example, compare the 040 field.  It has a total of 7 subfields: `$a $b $e $c $d $d $d` (3 subfield $d).  But the current template shows this as `$a $b $e $c $d` (1 subfield $d), with all `$d` values in the same subfield.

![040_good](https://user-images.githubusercontent.com/2213836/194186309-6290160d-ca1e-4ae3-9b75-7ae893880d20.png)

![040_bad](https://user-images.githubusercontent.com/2213836/194186325-9a0aa48c-6684-43ca-89bb-efa323d522a6.png)